doc/vic_installation/using_tls_with_vic.md

```diff
@@ -8,14 +8,21 @@ Familiarize yourself with the instructions for protecting the Docker daemon sock
 
 **Procedure**
 
-1. Use SSH to log in to the vSphere Integrated Containers appliance.
-2. Follow the instructions at https://docs.docker.com/engine/security/https/ to create TLS certificates on the virtual container host.
-3. Log into your Docker client.
-4. Run the following command to copy the certificate files into the `.docker` folder in your docker client. <pre>scp ca.pem server-cert.pem server-key.pem 
-root@<i>VIC_appliance_address</i>:/root/.docker/ 
-&& ssh root@vch /opt/dockerd.sh</pre>
-5. Restart the Docker daemon.
-
-After you restart the Docker daemon, Docker will use TLS authentication for all connections between the client and the virtual container host. 
-
+1. Use SSH to log in to the vSphere Integrated Containers appliance as `root`.
+2. Make sure that you are indeed logged in to the appliance as `root`. 
 
+ If you are logged in as user `tc`, change user to `root`.<pre>su root</pre>
```

Comment:
`sudo /bin/ash` may work better - that way you don't need to supply the password. I'm not sure what the password is for tc1